src/core/elf.c

```diff
@@ -3,10 +3,54 @@
 #include <mmu/paging.h>
 #include <string.h>
 
+#define ELF_TYPE_EXECUTABLE         0x2
```

Comment:
Shouldn't we move these constants to the header file? That would make me happier :)

Reply:
Done